test/functionTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+require('functions.php');
+use PHPunit\Framework\TestCase;
+
+class functionTest extends TestCase {
+
+    public function testSuccessloopCartoons() {
+        $expected = "<div class='eachCharacter'><h2 class='charName'>Test Name</h2><p class='cardContent'>From: Test Show</p><p class='cardContent'>Species: Test Species</p><p class='cardContent'>Catchphrase: Test Phrase</p><img src=Test Image></div>";
```

Comment:
maybe split this into a few lines just so it isn't so long?
```suggestion
        $expected = "<div class='eachCharacter'><h2 class='charName'>Test Name</h2>
        <p class='cardContent'>From: Test Show</p><p class='cardContent'>Species: Test Species</p>
        <p class='cardContent'>Catchphrase: Test Phrase</p><img src=Test Image></div>";
```

Reply:
I just tried, as it for readability makes sense, but when I retested the code, it came back as a failed :(